partner_duplicate_mgmt/models/res_partner.py

```diff
@@ -166,14 +173,14 @@ def create(self, vals):
     @api.multi
     def write(self, vals):
         res = super(ResPartner, self).write(vals)
-        if 'name' in vals or 'firstname' in vals or 'lastname' in vals:
-            self._update_indexed_name()
 
         if (
             'parent_id' in vals or 'name' in vals or
             'lastname' in vals or 'firstname' in vals or
             'company_type' in vals or 'is_company' in vals
         ):
+            self._update_indexed_name()
```

Comment:
looks to me like a bug fix that affects 10.0 as well ?

Reply:
No I don't think so, I can't remember now why i did this, but I'm sure that it was specific to 11.0. I'll think about it and justify my answer later.